data/json/items/gun/shot.json

```
@@ -640,9 +640,27 @@
     "copy-from": "shotgun_s",
     "type": "ITEM",
     "subtypes": [ "GUN" ],
-    "name": { "str": "double barrel shotgun" },
-    "description": "An old shotgun, possibly antique.  It is little more than a pair of barrels, a wood stock, and a hammer to strike the cartridges.",
+    "name": { "str": "double-barreled shotgun" },
+    "description": "The classic double-barreled shotgun, with the barrels arranged side-by-side.  It consists of little more than 2 barrels, a striker to hit the shell, and a wooden stock",
     "//": "Based on a CZ-USA Redhead 26in 12ga 3in ou",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "s&r",
+        "name": { "str": "S&R Ranger 12 gauge shotgun" },
+        "description": "An antique double-barreled shotgun chambered in 12 gauge.  These were made by many different manufacturers, but these inexpensive firearms were all sold in the Sears and Roebuck catalog in the days of yore."
+      },
+      {
+        "id": "pbt",
+        "name": { "str": "Parker Bros Trojan 12 gauge shotgun" },
+        "description": "An antique double-barreled shotgun chambered in 12 gauge.  Not to be confused with the board game company of the same name, Parker Bros.  made shotguns from 1867 to 1942, before they were phased out of production by Remington, who had acquired the company the previous decade.  These shotguns were sold in many different \"grades\" at varying price points, with higher quality of engravings being included at higher cost.  This particular shotgun is a \"Trojan\", which was the cheapest grade Parker Bros.  sold."
```

nonono, if the tests do weird, we fight it, not avoid it
```suggestion
        "name": { "str": "Parker Bros. Trojan 12 gauge shotgun", "//NOLINT(cata-text-style)": "Bros. is an abbreviation, not the end of sentence" },
        "description": { "str": "An antique double-barreled shotgun chambered in 12 gauge.  Not to be confused with the board game company of the same name, Parker Bros. made shotguns from 1867 to 1942, before they were phased out of production by Remington, who had acquired the company the previous decade.  These shotguns were sold in many different \"grades\" at varying price points, with higher quality of engravings being included at higher cost.  This particular shotgun is a \"Trojan\", which was the cheapest grade Parker Bros. sold.", "//NOLINT(cata-text-style)": "Bros. is an abbreviation, not the end of sentence" },
```